Modules/_operator.c

```diff
@@ -698,6 +698,20 @@ _operator_index(PyObject *module, PyObject *a)
     return PyNumber_Index(a);
 }
 
+/*[clinic input]
+_operator.is_none = _operator.neg
+
+Same as a is None.
+[clinic start generated code]*/
+
+static PyObject *
+_operator_is_none(PyObject *module, PyObject *a)
+/*[clinic end generated code: output=07159cc102261dec input=0448b38af7b8533d]*/
+{
+    PyObject *result = Py_IsNone(a) ? Py_True : Py_False;
+    return Py_NewRef(result);
```

Comment:
```suggestion
    return PyBool_FromLong(Py_IsNone(a));
```

Reply:
I think the same argument as above applies here: While I like your suggested implementation more, this would deviate from the rest in that file as the only differing implementation.
IMHO, this could be done in a separate refactoring PR along with "optimizing" the other implementations as well.
What's your opinion on that?

Reply:
Yes, we can totally go with this code, it is valid and it works. As I said, it is more like some nitpicks :)

Reply:
Thank you very much. I have opened a separate issue for all these things: #115830